topics/transcriptomics/tutorials/scrna-umis/tutorial.md

```diff
@@ -0,0 +1,553 @@
+
+One of the major issues with sequencing is that the read fragments require amplification before they can be sequenced. A gene with a single mRNA transcript will not be detected by most sequencers, so it needs to be duplicated 100-1000x times for the sequencer to 'see' it.
+
+Amplification is an imprecise process however, since some reads are amplified more than others, and subsequent amplification can lead to these over-amplified reads being over-amplified even more, leading to an exponential bias of some reads over others.
+
+## Naive Amplification
+
+ ![Amplification Bias](../../images/scrna_amplif_errors.svg "A cell with two reads from different transcripts being amplified unevenly")
+
+
+
+Consider the above example where two reads from different transcripts are amplified unevenly. The resulting frequency table would yield:
+
+> |  | Reads in Cell 1 |
+> |--|------------------|
+> | Gene Red | 5 |
+> | Gene Blue | 0 |
+{: .matrix}
+
+But the truth is entirely different (i.e. Gene Red should have 1 count, and Gene Blue should also have 1 count).
+How do we correct for this bias?
+
+## Amplification with UMIs
+
+**Unique Molecular Identifiers** (or *UMIs*) constitute the second portion of a barcode, where their role is to *uniquely* count reads such that amplicons of the same read are only counted once, e.g:
+
+ ![Amplification Bias with UMIs](../../images/scrna_amplif_errors_umis.svg "A cell with four unique transcripts, two from Gene Red and two from Gene Blue")
+
+ Here, we see two unique transcripts from Gene Red and two unique transcripts from Gene Blue, each given a (coloured) UMI. After amplificaiton, Gene Red has more reads than Gene Blue. If we were to construct a frequency table as before to count the reads, we would have:
+
+> |  | Reads in Cell 1 |
+> |--|-----------------|
+> | Gene Red | 6 |
+> | Gene Blue | 3 |
+{: .matrix}
+
+This information is false, because it shows that Red has twice the expression that Blue does. However, we can reconstitute the true count by considering the UMI information:
+
+> |  | UMI colour  | Reads in Cell 1 |
+> |--|-------------|-----------------|
+> | Gene Red | Pink | 2 |
+> |          | Blue | 4 |
+> | Gene Blue | Brown | 1 |
+> |           | Green | 2 |
+{: .matrix}
+
+From this we can then make the decision to ignore the frequencies of these UMIs, and simply count how many *unique* UMIs we see in each gene:
+
+> |  | Set of UMIs in Gene | UMIs in Cell 1 |
+> |--|---------------------|----------------|
+> | Gene Red | {Pink, Blue} | 2 |
+> | Gene Blue | {Brown, Green} | 2 |
+{: .matrix}
+
+This then provides us with the true count of the number of true transcripts for each gene as given by our original figure.
+
+> |  | UMIs in Cell 1 |
+> |--|----------------|
+> | Gene Red | 2 |
+> | Gene Blue | 2 |
+{: .matrix}
+
+
+> ### {% icon question %} Questions about UMIs
+> 
+> 1. Are UMIs specific to genes? i.e. Can the same UMI map to different genes?
```

Comment:
Isn't the answer to 1.Q1 no and to 1.Q2 yes.? i.e. in the beginning of a sentence is also unusal.